extension/src/shared/constants/api.ts

```diff
@@ -26,6 +26,11 @@ export const setGoogleStreetViewAPIKey = (key: string) => {
   GOOGLE_STREET_VIEW_API_KEY = key;
 };
 
+export let HIDE_FEEDBACK: boolean | undefined;
+export const setHideFeedback = (hide: boolean) => {
+  HIDE_FEEDBACK = hide;
```

Comment:
BTW in a simple case, we might be able to use jotai's atom to update the state without reloading browser.